src/controls/dimension-slider.js

```diff
@@ -59,9 +76,7 @@ export default class DimensionSlider extends EventSubscriber {
      * @type {Array.<string,function>}
      */
     sub_list = [[IMAGE_CONFIG_UPDATE,
-                    (params = {}) => this.onImageConfigChange(params)],
-                [IMAGE_VIEWER_RESIZE,
```

Comment:
I believe there is also a an import of the constant left still (IMAGE_VIEWER_RESIZE)